drivers/ethernet/eth_enc28j60.c

```diff
@@ -360,6 +491,14 @@ static int eth_enc28j60_init(struct device *dev)
 		return -EINVAL;
 	}
 
+	/* Initialize GPIO */
+	if (gpio_pin_configure(enc_spi_cs.gpio_dev, config->spi_cs_pin,
+			       (GPIO_DIR_OUT | GPIO_PUD_PULL_DOWN))) {
+		SYS_LOG_ERR("Unable to configure GPIO pin %u",
+			    config->gpio_pin);
+		return -EINVAL;
+	}
+
```

Comment:
You don't need to configure the gpio cs by yourself, neither in spi legacy nor the new one.